sequencer/src/persistence/sql.rs

```diff
@@ -2000,6 +2000,43 @@ impl SequencerPersistence for Persistence {
             .transpose()
     }
 
+    async fn store_eqc(
+        &self,
+        high_qc: QuorumCertificate2<SeqTypes>,
+        next_epoch_high_qc: NextEpochQuorumCertificate2<SeqTypes>,
+    ) -> anyhow::Result<()> {
+        let eqc_bytes =
+            bincode::serialize(&(high_qc, next_epoch_high_qc)).context("serializing eqc")?;
```

Comment:
I would prefer if there is separate column for each certificate in the SQL table